internal/support/notifications/config/config.go

```diff
@@ -40,12 +40,17 @@ type WritableInfo struct {
 
 type SmtpInfo struct {
 	Host                 string
-	Username             string
-	Password             string
+	Username             string // deprecated in V2
+	Password             string // deprecated in V2
```

Comment:
Remove

Reply:
we plan to remove them after https://github.com/edgexfoundry/edgex-go/pull/3492 merged, or it would cause V1 code failure

Reply:
Got it.